GVRf/Framework/src/org/gearvrf/GVRViewManager.java

```diff
@@ -312,7 +311,18 @@ public void beforeDrawEyes() {
                 getMainScene().addSceneObject(mSplashScreen);
             }
 
-            mScript.onInit(GVRViewManager.this);
+            try {
+                mScript.onInit(GVRViewManager.this);
+            } catch (Throwable t) {
+                t.printStackTrace();
```

Comment:
Hmm? printStackTrace() gives the full stack trace, with the right pid. Maybe you can miss it if you are searching for tag=="GVRF" (but why would you do that?) but it's there when you search by process id.